includes/gating/functions.php

```diff
@@ -96,11 +96,11 @@ function get_valid_gating_types() {
 function maybe_add_padlock_to_title( string $title, int $id = 0 ) : string {
 
 	// If no explicit post ID passed, try to grab implicit one
-	if( empty( $id ) ){
+	if ( empty( $id ) ) {
 		$id = get_the_ID();
 
 		// No post ID found. Assume no padlock.
-		if( empty( $id ) ) {
+		if ( empty( $id ) ) {
```

Comment:
I think we can use a ternary operator here to deal with both cases, like:

```
$id = ( empty( $id ) ? get_the_ID() : $id );

if ( empty( $id ) ) {
     return $title;
}
```

That'll save using two if statements

Reply:
No problem.